modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resources_list3.html

```diff
@@ -0,0 +1,32 @@
+<section id="dataset-resources" class="resources">
+  <h3>{{ list_title or _('Resources') }} ({{ resources|count }})</h3>
+
+  {% block resource_list %}
+  {% if resources %}
+    <ul class="resource-list resource-list-apiset">
+      {% block resource_list_inner %}
+      {% for resource in resources %}
+        {% snippet 'package/snippets/resource_item3.html', pkg=pkg, res=resource, draft=draft or false, resource_under_edit=resource_under_edit or false %}
+      {% endfor %}
+      {% endblock %}
+    </ul>
+  {% else %}
+    <p class="empty">
+      {% if h.check_access('package_update', {'id':pkg.id }) %}
+        {# Comment out "add some" as action doesn't exist yet #}
```

Comment:
Is this comment here for a purpose or should it be removed?

Reply:
it has been removed